frontend/catalyst/api_extensions/quantum_operators.py

```diff
@@ -153,6 +153,8 @@ def circuit():
         in_classical_tracers=in_classical_tracers,
         out_classical_tracers=[m],
         regions=[],
+        reset=reset,
+        postselect=postselect,
```

Comment:
I'm curious why not just append `reset` to `in_classical_tracers` and extract it in the constructor? `postselect` is already there as well. Or just remove `postselect` from `in_classical_tracers`.

Reply:
I have not given it much thought, I would say at this point this can be cleaned up as you suggest. Will do.